app/services/marketing_site.rb

```diff
@@ -1,6 +1,12 @@
 class MarketingSite
   BASE_URL = URI('https://www.login.gov').freeze
 
+  HELP_CENTER_ARTICLES = {
+    'verify-your-identity' => [
+      'how-to-add-images-of-your-state-issued-id',
+    ].freeze,
+  }.freeze
```

Comment:
I think for ease of searching and copy-pasting, it would be easier to have an array of strings that we process?

This might be a little too messy/clever... but it would have the same result

```suggestion
  # builds a Hash<String, Array<String>> of { category => [article, article] }
  HELP_CENTER_ARTICLES = %w[
    /verify-your-identity/how-to-add-images-of-your-state-issued-id
  ].map { |path| path.split('/') }.
  reduce(Hash.new([])) do |hsh, (_, category, article)|
    hsh[article] += category
    hsh
  end.freeze
```

Reply:
Hm, yeah, or maybe we could just have the array of full paths, and query against that 🤔 

```rb
HELP_CENTER_ARTICLES = %w[
  /verify-your-identity/how-to-add-images-of-your-state-issued-id
]

def self.valid_help_center_article?(category:, article:)
  HELP_CENTER_ARTICLES.include?("#{category}/#{article}")
end

# or:
#
# def self.valid_help_center_article?(path)
#   HELP_CENTER_ARTICLES.include?(path)
# end
```

Reply:
Oh great point, I like the `category:, article:`one (might need a leading slash on there) because it minimizes the chance of a missing trailing slash or something messing things up

Reply:
and maybe add a `.to_set` on there too

Reply:
Updated in 99b0b31.